client/api/src/execution_extensions.rs

```diff
@@ -90,14 +104,20 @@ impl<Block: traits::Block> ExecutionExtensions<Block> {
 		keystore: Option<BareCryptoStorePtr>,
 	) -> Self {
 		let transaction_pool = RwLock::new(None);
-		Self { strategies, keystore, transaction_pool }
+		let extensions_maker = Box::new(());
```

Comment:
```suggestion
		let extensions_maker = RwLock::new(Box::new(()));
		Self { strategies, keystore, extensions_maker, transaciton_pool }
```
to avoid repetitions

Reply:
Had that, Rust complained and couldn't understand the type given was okay. Will try again with latest rust stable.